server/src/main/java/module-info.java

```diff
@@ -297,6 +297,7 @@
     exports org.elasticsearch.plugins;
     exports org.elasticsearch.plugins.interceptor to org.elasticsearch.security, org.elasticsearch.serverless.rest;
     exports org.elasticsearch.plugins.spi;
+    exports org.elasticsearch.readiness;
```

Comment:
Tests don't run modularized, so why do we need to add this export?

Reply:
Because I did not know that :)